File: output/ChatGPT/avatar/Java/C++/atcoder_ABC127_B.cpp
#include <iostream>
#include <cstdio>
#include <cstring>
#include <sstream>
#include <stdexcept>

class SimpleScanner {
private:
    static const int BUFFER_SIZE = 10240;
    std::istream& in;
    char buffer[BUFFER_SIZE];
    int bufferPos;
    int bufferSize;
    bool eof;

    char read() {
        if (bufferPos >= bufferSize) {
            bufferPos = 0;
            in.read(buffer, BUFFER_SIZE);
            bufferSize = in.gcount();
            if (bufferSize <= 0) {
                eof = true;
                return '\0';
            }
        }
        return buffer[bufferPos++];
    }

    void checkEof() {
        if (eof) {
            throw std::out_of_range("End of input");
        }
    }

public:
    SimpleScanner(std::istream& in) : in(in), bufferPos(0), bufferSize(0), eof(false) {}

    char nextChar() {
        checkEof();
        char c = read();
        checkEof();
        return c;
    }

    std::string next() {
        char c;
        do {
            c = read();
            checkEof();
        } while (std::isspace(c));
        std::stringstream ss;
        do {
            ss << c;
            c = read();
        } while (!eof && !std::isspace(c));
        return ss.str();
    }

    int nextInt() {
        std::string s = next();
        return std::stoi(s);
    }

    long long nextLong() {
        std::string s = next();
        return std::stoll(s);
    }

    double nextDouble() {
        std::string s = next();
        return std::stod(s);
    }
};

int main() {
    SimpleScanner scanner(std::cin);
    int r = scanner.nextInt();
    int d = scanner.nextInt();
    long long x = scanner.nextLong();
    for (int i = 0; i < 10; ++i) {
        x = r * x - d;
        std::cout << x << std::endl;
    }
    return 0;
}

File: output/ChatGPT/avatar/Java/Python/atcoder_ABC127_B.py
import sys

class SimpleScanner:
    def __init__(self, file):
        self.file = file
        self.buffer = ''
        self.eof = False
    
    def read(self):
        if len(self.buffer) == 0:
            self.buffer = self.file.readline().strip().split()
            if len(self.buffer) == 0:
                self.eof = True
                return ''
        return self.buffer.pop(0)
    
    def check_eof(self):
        if self.eof:
            raise ValueError('No more elements to read.')
    
    def next_char(self):
        self.check_eof()
        char = self.read()
        self.check_eof()
        return char
    
    def next(self):
        char = self.read()
        while char.isspace():
            char = self.read()
        word = char
        char = self.read()
        while not self.eof and not char.isspace():
            word += char
            char = self.read()
        return word
    
    def next_int(self):
        return int(self.next())
    
    def next_long(self):
        return int(self.next())
    
    def next_double(self):
        return float(self.next())

def main():
    r = scanner.next_int()
    d = scanner.next_int()
    x = scanner.next_long()
    for _ in range(10):
        x = r * x - d
        print(x)

scanner = SimpleScanner(sys.stdin)
main()